internal/painter/gl/draw.go

```diff
@@ -231,6 +369,66 @@ func (p *painter) lineCoords(pos, pos1, pos2 fyne.Position, lineWidth, feather f
 	}, halfWidth, featherWidth
 }
 
+/*
+func (p *painter) flexLineCoordsNew(pos, pos1, pos2 fyne.Position, lineWidth, feather float32, frame fyne.Size) ([]float32, float32, float32) {
```

Comment:
We prefer to not have dead code. Would you mind removing this until it is of use?